Makefile

```diff
@@ -56,10 +56,14 @@ HOSTMOUNT_PREFIX ?= /
 KUBECONFIG ?= ${HOME}/.kube/config
 E2E_TEST_CONFIG ?=
 E2E_PULL_IF_NOT_PRESENT ?= false
+E2E_TEST_FULL_IMAGE ?= false
```

Comment:
Please enable this in `scripts/test-infra/test-e2e.sh` (as commented [here](https://github.com/kubernetes-sigs/node-feature-discovery/pull/1489#discussion_r1419200447))